### ISSUE TITLE
[ubuntu] libudev.so.0: cannot open shared object file

### ISSUE THREAD

**Issue by krazyjakee** [oklai/koala#76]
When trying to run koala in linux I get this error:

```
error while loading shared libraries: libudev.so.0: cannot open shared object file: No such file or directory
```

Here is the fix:

Create global symlink to `libudev.so.1` by hand
install the package `libudev1`, and there is `libudev.so.1` at `/lib/x86_64-linux-gun/libudev.so.1`. On Ubuntu for example, Run:

``` bash
apt-get install libudev1
cd /lib/x86_64-linux-gnu/
ln -s libudev.so.1 libudev.so.0
```

Maybe it would be a good idea to upgrade to libudev.so.1.

**Comment by Agneli** [comment]
Thank you very much @krazyjakee.
Your solution fixes a runtime error of Popcorn-time on Debian 7 64bit.

**Comment by yasomal** [comment]
Thank you very much @krazyjakee.
It fixed Koala loading error on my Linux Mint 17.2! :+1:

**Comment by BlackHole1** [comment]
thx! :+1:

**Comment by silicon755** [comment]
Thank you very much @krazyjakee 
I managed to fix Koala loading error on my linux kali 2.0 64bit.
